src/dlgProfilePreferences.cpp

```diff
@@ -438,7 +438,14 @@ void dlgProfilePreferences::initWithHost(Host* pHost)
     //checkBox_LF_ON_GA->setChecked( pHost->mLF_ON_GA );
     checkBox_mUSE_FORCE_LF_AFTER_PROMPT->setChecked(pHost->mUSE_FORCE_LF_AFTER_PROMPT);
     USE_UNIX_EOL->setChecked(pHost->mUSE_UNIX_EOL);
+
+    checkBox_discordGameAddress->setChecked(pHost->mDiscordHideAddress);
```

Comment:
Could put these in a groupbox with a checkbox to allow user to enable/disable Discord usages (and disable and hide it on platforms where Discord is not available)...